composer.json

```diff
@@ -20,7 +20,8 @@
         "phpunit/phpunit": "~6.0",
         "psr/container": "~1.0",
         "pimple/pimple": "~3.0",
-        "symfony/yaml": "~2.7|~3.0"
+        "symfony/yaml": "~2.7|~3.0",
+        "doctrine/common": "^2.8.1"
```

Comment:
@TiMESPLiNTER please use lowest supported version, not the newest once

Reply:
Done.